.github/workflows/ci.yaml

```diff
@@ -7,7 +7,7 @@ name: CI
 
 jobs:
   rust_tests:
-    name: Run tests
+    name: Run rust tests
     runs-on:  ${{ matrix.os }}
```

Comment:
Do we need to run Rust tests on all OSs? In theory we wouldn't be testing any platform specific behavior.

Reply:
I agree that we currently do not test anything OS specific but that can change...